Server/src/controllers/GraphsController.ts

```diff
@@ -16,8 +16,8 @@ export class GraphsController {
      * @param response 
      * An object containing a response: Response
      */
-    async createRequestForSingleSavedGraph(request: Request, response: Response) {
-        let requestParam = request.params.oneSavedGraph;
+    async createRequestForSingleGraph(request: Request, response: Response) {

```

Comment:
i appreciate you taking the time to clean up the code, shorter concise functions are easier to work with